src/views/v2/recent-past.php

```diff
@@ -71,9 +71,9 @@
 
 		</div>
 
-		<?php $this->template( 'components/after' ); ?>
+		<?php //$this->template( 'components/after' ); ?>
 
 	</div>
 </div>
 
-<?php $this->template( 'components/breakpoints' ); ?>
+<?php //$this->template( 'components/breakpoints' ); ?>
```

Comment:
* :warning: This comment is 67% valid code; is this commented out code?
* :no_entry_sign: No space found before comment text; expected "// $this->template( 'components/breakpoints' );" but found "//$this->template( 'components/breakpoints' );"

<sub>hash: 21861d21a03931ac08eb503f54152b5c</sub>